From: Rod Whitby <rod@whitby.id.au>
Subject: Native little endian kernel support

There is no need for Apex to swap kernel endianess on these devices.

Signed-off-by: Rod Whitby <rod@whitby.id.au>
--

Index: apex/src/mach-ixp42x/slugos-dsmg600-arm_config
===================================================================
--- apex.orig/src/mach-ixp42x/slugos-dsmg600-arm_config	2008-05-28 10:52:50.000000000 +0930
+++ apex/src/mach-ixp42x/slugos-dsmg600-arm_config	2008-05-28 10:52:58.000000000 +0930
@@ -132,7 +132,7 @@
 CONFIG_AUTOBOOT=y
 CONFIG_AUTOBOOT_DELAY=10
 CONFIG_ENV_STARTUP_KERNEL_COPY=y
-CONFIG_ENV_REGION_KERNEL_SWAP=y
+# CONFIG_ENV_REGION_KERNEL_SWAP is not set
 # CONFIG_ENV_STARTUP_PREFIX_P is not set
 
 #
Index: apex/src/mach-ixp42x/slugos-fsg3-arm_config
===================================================================
--- apex.orig/src/mach-ixp42x/slugos-fsg3-arm_config	2008-05-28 10:52:50.000000000 +0930
+++ apex/src/mach-ixp42x/slugos-fsg3-arm_config	2008-05-28 10:52:58.000000000 +0930
@@ -132,7 +132,7 @@
 CONFIG_AUTOBOOT=y
 CONFIG_AUTOBOOT_DELAY=10
 CONFIG_ENV_STARTUP_KERNEL_COPY=y
-CONFIG_ENV_REGION_KERNEL_SWAP=y
+# CONFIG_ENV_REGION_KERNEL_SWAP is not set
 # CONFIG_ENV_STARTUP_PREFIX_P is not set
 
 #
Index: apex/src/mach-ixp42x/slugos-nas100d-arm_config
===================================================================
--- apex.orig/src/mach-ixp42x/slugos-nas100d-arm_config	2008-05-28 10:52:56.000000000 +0930
+++ apex/src/mach-ixp42x/slugos-nas100d-arm_config	2008-05-28 10:52:58.000000000 +0930
@@ -132,7 +132,7 @@
 CONFIG_AUTOBOOT=y
 CONFIG_AUTOBOOT_DELAY=10
 CONFIG_ENV_STARTUP_KERNEL_COPY=y
-CONFIG_ENV_REGION_KERNEL_SWAP=y
+# CONFIG_ENV_REGION_KERNEL_SWAP is not set
 # CONFIG_ENV_STARTUP_PREFIX_P is not set
 
 #
Index: apex/src/mach-ixp42x/slugos-nslu2-16mb-arm_config
===================================================================
--- apex.orig/src/mach-ixp42x/slugos-nslu2-16mb-arm_config	2008-05-28 10:52:50.000000000 +0930
+++ apex/src/mach-ixp42x/slugos-nslu2-16mb-arm_config	2008-05-28 10:52:58.000000000 +0930
@@ -134,7 +134,7 @@
 CONFIG_AUTOBOOT=y
 CONFIG_AUTOBOOT_DELAY=10
 CONFIG_ENV_STARTUP_KERNEL_COPY=y
-CONFIG_ENV_REGION_KERNEL_SWAP=y
+# CONFIG_ENV_REGION_KERNEL_SWAP is not set
 # CONFIG_ENV_STARTUP_PREFIX_P is not set
 
 #
Index: apex/src/mach-ixp42x/slugos-nslu2-arm_config
===================================================================
--- apex.orig/src/mach-ixp42x/slugos-nslu2-arm_config	2008-05-28 10:52:50.000000000 +0930
+++ apex/src/mach-ixp42x/slugos-nslu2-arm_config	2008-05-28 10:52:58.000000000 +0930
@@ -134,7 +134,7 @@
 CONFIG_AUTOBOOT=y
 CONFIG_AUTOBOOT_DELAY=10
 CONFIG_ENV_STARTUP_KERNEL_COPY=y
-CONFIG_ENV_REGION_KERNEL_SWAP=y
+# CONFIG_ENV_REGION_KERNEL_SWAP is not set
 # CONFIG_ENV_STARTUP_PREFIX_P is not set
 
 #
